Tests/DatadogTests/Datadog/RUM/Instrumentation/Resources/URLSessionRUMResourcesHandlerTests.swift

```diff
@@ -49,7 +49,7 @@ class URLSessionRUMResourcesHandlerTests: XCTestCase {
             ]
         )
 
-        XCTAssertEqual(request.value(forHTTPHeaderField: "x-datadog-origin"), "rum")
+        XCTAssertEqual(request.value(forHTTPHeaderField: TracingHTTPHeaders.originField), "rum")
         XCTAssertEqual(request.value(forHTTPHeaderField: TracingHTTPHeaders.traceIDField), "1")
```

Comment:
suggestion/ The fact that we don't have to revamp this test after implemented change:
* from setting `x-datadog-origin` only in `.datadog` 
* to setting it for all header types (`.datadog`, `.b3`, ... `tracecontext`) 

means that this test is missing that coverage. 

Instead of having a single test that sets all header types and asserts on `x-datadog-origin` presence, we should define separate test for `x-datadog-origin`. With the recent change, it should assert that this header is set for all header types.